Bottom: baeaa6770a2ee835729fd4c140486ef50f183ede
Top:    397ec5fc1a63ecb4bf9834bcc1ff57ac4aee7f3e
Author: Peter Clifton <peter@clifton-electronics.co.uk>
Date:   2016-12-04 15:56:08 +0000

Remove left over debug


---

diff --git a/src/hid/common/object3d.c b/src/hid/common/object3d.c
index 7039457..55571cf 100644
--- a/src/hid/common/object3d.c
+++ b/src/hid/common/object3d.c
@@ -1251,8 +1251,6 @@ object3d_from_copper_layers_within_area (POLYAREA *area)
 #endif
                                 copper_appearance,
                                 NULL));
-
-      //break;
     }
